Example/Source/Examples/Interactive/InsertBrickViewController.swift

```diff
@@ -21,23 +23,39 @@ class InsertBrickViewController: BrickApp.BaseBrickController, HasTitle {
         return "Shows different ways of inserting a brick"
     }
 
-    var selectedSegmentIndex: Int = 0 {
+    var selectedSegmentIndex: Int = 0
+    var titles: [String] = []
+
+    var selectedAppearSegmentIndex: Int = 0 {
         didSet {
             let appearBehavior: BrickAppearBehavior?
-            switch selectedSegmentIndex {
+            switch selectedAppearSegmentIndex {
             case 1: appearBehavior = BrickAppearTopBehavior()
             case 2: appearBehavior = BrickAppearBottomBehavior()
+            case 3: appearBehavior = BrickAppearFadeBehavior()
             default: appearBehavior = nil
             }
             self.layout.appearBehavior = appearBehavior
         }
     }
 
-    var titles: [String] {
-        return ["No animation", "From top", "From bottom"]
+    var appearTitles: [String] {
+        return ["No animation", "From top", "From bottom", "Fade in"]
+    }
+
+    var insertToIndex: Int {
```

Comment:
This variable name is a bit confusing, feels more like a function name. Can you call it more `indexToInsertTo`?